src/components/Users/UserSection.jsx

```diff
@@ -21,14 +27,28 @@ const UserSection = ({ loadingUsers, loadingActive }) => {
     <>
       <ManageUsers />
       {loadingUsers ? (
-        <section className="panel">
-          <strong>Users List</strong>
-          <br />
-          <br />
-          <div style={{ marginBottom: '20px', textAlign: 'center' }}>Loading users list...</div>
-        </section>
-      ) : (
         <UsersList loadingActive={loadingActive} />
```

Comment:
Minor thing, I think I can update this. But the ordering of this ternary should be reversed.

If loadingUsers is true, we should have the component with the loading animation and not UserList.

Reply:
Fixed the ordering, looks good now.